components/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/OAuthConstants.java

```diff
@@ -623,6 +623,15 @@ public static class OIDCConfigProperties {
         public static final String TOKEN_REVOCATION_WITH_IDP_SESSION_TERMINATION =
                 "tokenRevocationWithIDPSessionTermination";
         public static final String TOKEN_BINDING_VALIDATION = "tokenBindingValidation";
+        public static final String USE_CLIENT_ID_AS_SUB_CLAIM_FOR_APP_TOKENS = "useClientIdAsSubClaimForAppTokens";
+        public static final boolean USE_CLIENT_ID_AS_SUB_CLAIM_FOR_APP_TOKENS_OLD_APP_DEFAULT_VALUE = false;
+        public static final boolean USE_CLIENT_ID_AS_SUB_CLAIM_FOR_APP_TOKENS_NEW_APP_DEFAULT_VALUE = true;
+        public static final String OMIT_USERNAME_IN_INTROSPECTION_RESP_FOR_APP_TOKEN =
+                "omitUsernameInIntrospectionRespForAppTokens";
+        public static final boolean OMIT_USERNAME_IN_INTROSPECTION_RESP_FOR_APP_TOKEN_OLD_APP_DEFAULT_VALUE =
```

Comment:
Why we need this?

Reply:
Need to remove. This was used earlier, then replaced usage with parse boolean. 
Removed with a52e255e372f6063561a377b037be2b0307f83f4